jicoco/src/main/java/org/jitsi/xmpp/mucclient/MucClient.java

```diff
@@ -281,24 +285,6 @@ public void connectionClosedOnError(Exception e)
             {
                 logger.warn("Closed on error:", e);
             }
-
-            @Override
-            public void reconnectionSuccessful()
-            {
-                logger.info("Reconnection successful.");
-            }
-
-            @Override
-            public void reconnectingIn(int i)
-            {
-                logger.info("Reconnecting in " + i);
-            }
-
-            @Override
-            public void reconnectionFailed(Exception e)
-            {
-                logger.warn("Reconnection failed: ", e);
-            }
         });
```

Comment:
I suppose these are gone in latest API? There is no option to have that information back?
I remember these were useful when debugging some reconnects ...

Reply:
Ah, it's been moved to a separate listener object, I've added it.